node_scripts/install/install.py

```diff
@@ -27,6 +27,11 @@ def setup_node():
         plugins.setup_plugins(is_master=True, is_worker=True)
         scripts.run_custom_scripts(is_master=True, is_worker=True)
 
+        #Write sentinel file on disk for host to know this is the master
+        path = os.environ['PWD'] + '/MASTER'
+        print('Writing master file to: {}'.format(path))
+        f = open(path, 'w')
+        f.write(master_node.ip_address)
```

Comment:
This is already done, here:https://github.com/Azure/aztk/blob/c244e3437b74fa899e6b7450f642e3b165da0903/node_scripts/install/spark.py#L50

Reply:
Removed